en/app.ftl

```diff
@@ -330,6 +330,22 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
 
 premium-promo-pricing-free-price = Your current Free plan
 
+## The Premium waitlist page
+
+waitlist-heading = Join the { -brand-name-relay-premium } Waitlist
+waitlist-lead = Get notified when { -brand-name-firefox-relay-premium } is available for your region.
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+waitlist-control-email-placeholder = yourname@example.com
```

Comment:
Does it matter whether this is translated or not? If it is, let's add a comment.
See examples from mozilla.org: https://transvision.mozfr.org/?recherche=yourname%40example.com&repo=mozilla_org&sourcelocale=en&locale=de&search_type=strings_entities

Reply:
I'm not actually sure what the best practice is here? The main thing to demonstrate is that it takes an email address, so anything of that general form should work, but I can imagine that localisation is still useful - with the caveat that example.com is an actual example domain, for which I don't think there are local equivalents?

Reply:
Please add a comment explaining that example.com should not be translated, only `yourname`. As you said, that's a IANA domain that is safe to use, unlike translated ones (there are only a couple of exceptions that I'm aware of, but not worth calling out).

Reply:
Done in 5c3ee31.